lib/cmdlib/instance_migration.py

```diff
@@ -916,6 +917,10 @@ def _ExecMigration(self):
 
     self.feedback_fn("* starting memory transfer")
     last_feedback = time.time()
+
+    migration_caps = \
+      self.instance.hvparams.get(constants.HV_KVM_MIGRATION_CAPS, None)
```

Comment:
Good call. hvparams[HV_KVM_MIGRATION_CAPS] is a ':' delimited string, so I'll have it default to an empty string, which should serve the same purpose.